test/e2e/common/util.go

```diff
@@ -1084,9 +1094,38 @@ func DeletePools() {
 	}
 }
 
+func DeleteAllPoolFinalizers() {
+	poolGVR := schema.GroupVersionResource{
+		Group:    "openebs.io",
+		Version:  "v1alpha1",
+		Resource: "mayastorpools",
+	}
+
+	pools, err := gTestEnv.DynamicClient.Resource(poolGVR).Namespace("mayastor").List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		// This function may be called by AfterSuite by uninstall test so listing MSVs may fail correctly
+		logf.Log.Info("DeleteAllPoolFinalisers: list MSPs failed.", "Error", err)
+	}
+	if err == nil && pools != nil && len(pools.Items) != 0 {
```

Comment:
nit: you could argue the len check is not needed here

Reply:
Done